packages/firebase_crashlytics/firebase_crashlytics/test/firebase_crashlytics_test.dart

```diff
@@ -223,9 +221,10 @@ void main() {
 
       test('without character index', () async {
         final List<String> lines = <String>[
-          'package:flutter/src/widgets/framework.dart 3825  StatefulElement.build'
+          '#0      StatefulElement.build (package:flutter/src/widgets/framework.dart:3825:27)'
```

Comment:
I have some doubts about this change.
My understanding is that this test is supposed to check the case without the `:27`. Otherwise it is identical to `with character index`

Although to begin with, I'm not sure when this case happens. Any idea? I'm fine with just removing the test.

Reply:
Thanks for the review.

Column may not be included in stack trace in some cases. Because [there are if statements in stack trace body print function](https://github.com/dart-lang/sdk/blob/master/runtime/vm/object.cc#L24881).

The old implementation uses `Trace` API to parse stack trace but it converts to string then reparses it. This implementation uses `Trace` API directly.

Since all parsing is done with `Trace` API, parsing is no longer related to crashlytics package.

I will remove this test and replace `with character index` to `with symbolic stack trace`.